### ISSUE TITLE
Last stage

### ISSUE THREAD

**Issue by EvaceLi** [THUKey/pytorch-mobilenet-v3#5]
In original paper paper ,There is no activation function after avg_Pool　in fig5, but your code is added. Does this have any effect?

**Comment by ujsyehao** [comment]
you can see Table 1 or Tabel2, it exists activate after pool

**Comment by EvaceLi** [comment]
yes, it exists activate after pool in table1 or tabel2, but it not exists in fig5,Which one should we follow?

**Comment by comway-li** [comment]
> you can see Table 1 or Tabel2, it exists activate after pool

**Comment by kuan-wang** [comment]
The paper updated once on 14 May, but still has some bugs.  I think there should not be non-linear after pool.

**Comment by EvaceLi** [comment]
@kuan-wang 
thank you very much
